File: src/main/java/com/fengsheng/handler/select_role_tos.java
package com.fengsheng.handler;

import com.fengsheng.HumanPlayer;
import com.fengsheng.protos.Fengsheng;

public class select_role_tos extends AbstractProtoHandler<Fengsheng.select_role_tos> {
    @Override
    protected void handle0(HumanPlayer r, Fengsheng.select_role_tos pb) {
        r.getGame().tryContinueResolveProtocol(r, pb);
    }
}

File: src/main/kotlin/handler/SelectRoleTos.kt
package com.fengsheng.handler

import com.fengsheng.HumanPlayer
import com.fengsheng.protos.Fengsheng

class SelectRoleTos : AbstractProtoHandler<Fengsheng.select_role_tos>() {
    override fun handle0(r: HumanPlayer, pb: Fengsheng.select_role_tos) {
        r.game!!.tryContinueResolveProtocol(r, pb)
    }
}
